src/api/operator/types.ts

```diff
@@ -19,7 +19,14 @@ export interface FeeInformation {
 
 export type OrderKind = 'sell' | 'buy'
 
-export type OrderStatus = 'open' | 'filled' | 'canceled' | 'expired'
+export type OrderStatus = 'open' | 'filled' | 'cancelled' | 'expired' | 'signature pending'
+export enum RawOrderStatusFromAPI {
```

Comment:
Any reason to make it an `enum`?

Reply:
No, I have changed it to similar to previous one.